### DISCUSSION TITLE
如何在开启中文输入法的情况下快速输入[[

### DISCUSSION THREAD

**Discussion by lakako** [fcitx/fcitx5#986] [Q&A] [answered]
使用小鹤双拼，默认情况下，点击 `[` 会弹出输入框，可以选择多种不同的中括号

在实际使用中，很少连着使用两个全角中括号(【 【 )

有没有什么办法可以在开启中文输入法的情况下`双击[[`直接输入`[[`?

**Comment by wengxt** [comment, marked as answer]
你可以改标点设置里 [ ] 的映射
